docs/products/clickhouse/howto/create-dictionary.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Create dictionaries in Aiven for ClickHouse®
+sidebar_label: Create dictionary
+---
+
+Create dictionaries in Aiven for ClickHouse® to accelerate queries for better efficiency and performance.
+
+## Dictionaries in Aiven for ClickHouse
+
+A dictionary is a key -> attribute mapping useful for low latency lookup queries, when
```

Comment:
Following the style guide and for better accessibility, I'd use a hyphen here instead. 

```suggestion
A dictionary is a key-attribute mapping useful for low latency lookup queries, when
```